composer/loggers/wandb_logger.py

```diff
@@ -335,4 +354,36 @@ def _convert_to_wandb_image(image: Union[np.ndarray, torch.Tensor], channels_las
     if not channels_last:
         assert isinstance(image, np.ndarray)
         image = image.transpose(1, 2, 0)
-    return image
+    return np.asarray(image)
```

Comment:
I think this conversion should not be necessary because the top of this function converts it to numpy?